tests/v2/08_full_workflow/test_full_workflow_local_experimental_v2.py

```diff
@@ -0,0 +1,162 @@
+import pytest
```

Comment:
Quick check: is this module identical to the `local` one, apart from the definition of the backend?